bugwarrior/services/gitlab.py

```diff
@@ -121,6 +121,18 @@ class GitlabIssue(Issue):
     def _normalize_label_to_tag(self, label):
         return re.sub(r'[^a-zA-Z0-9]', '_', label)
 
+    # Override the method from parent class
+    def get_priority(self):
+        type_str = self.extra['type']
+        default_priority = self.origin['default_priority']
+        if type_str == 'todo':
+            return self.origin['todo_priority']
+        elif type_str == 'merge_request':
+            return self.origin['mr_priority']
+        elif type_str == 'issue':
+            return self.origin['issue_priority']
```

Comment:
Personally I'd find the following cleaner:

```py
default_priority_map = {
    'todo': self.origin['todo_priority'],
    'merge_request': self.origin['mr_priority'],
    'issue': self.origin['issue_priority']}
return default_priority_map.get(type_str, default_priority)
```

Reply:
Addressed in ef92fea968b42abf512002be4f56bd5463054e3a